8.1/cli/bullseye/entrypoint.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
As far as I can see, this file does not exist for any other version. Why is it necessary for PHP 8.1?

Reply:
removed